packages/lib/modules/vebal/vote/Votes/MyVotes/MyVotes.tsx

```diff
@@ -38,7 +39,10 @@ export function MyVotes() {
   const { isOpen, onOpen, onClose } = useDisclosure()
   const { isConnected } = useUserAccount()
 
-  const { loading: vebalUserDataLoading, myVebalBalance, noVeBalBalance } = useVebalUserData()
+  const { isLoading: vebalUserDataLoading, veBALBalance, noVeBALBalance } = useVebalUserData()
+
+  // FIXME: [JUANJO] everything needs a number here, but we should work with bigint
```

Comment:
Wherever numbers are used for calculation we need to change to use either bigint or bignumber, but we should probably have some tests around to refactor this.